data/api/client-server/room_state.yaml

```diff
@@ -20,8 +20,8 @@ paths:
     put:
       summary: Send a state event to the given room.
       description: |
-        State events can be sent using this endpoint.  These events will be
-        overwritten if `<room id>`, `<event type>` and `<state key>` all
+        State events can be sent using this endpoint.  These events will be appended
```

Comment:
appended to what?

Reply:
+1 we should not prescribe how the events are processed internally. I changed the wording from appended to "processed"